subsys/net/openthread/rpc/Kconfig

```diff
@@ -59,6 +59,18 @@ config OPENTHREAD_RPC_CLIENT_NUM_SENT_COAP_REQUESTS
 
 endmenu # "OpenThread over RPC client configuration"
 
+menu "OpenThread over RPC client configuration"
+	depends on OPENTHREAD_RPC_SERVER
+
+	config OPENTHREAD_RPC_MESSAGE_POOL
+		int "Maximum number of messages that can be allocated on server"
+		default 8
+		help
+		  Defines maximum number of messages that can be allocated on server at
```

Comment:
In fact, this is the maximum number of otMessage keys/handles that are passed to the RPC client and you can allocate many more otMessages on the server. Can this be rephrased to make it clear?